spec/react_on_rails/configuration_spec.rb

```diff
@@ -72,6 +72,18 @@ module ReactOnRails
       end
     end
 
+    describe ".build_production_command" do
+      it "if configured, ENV[\"WEBPACKER_PRECOMPILE\"] gets set to \"false\"" do
+        expect(ENV["WEBPACKER_PRECOMPILE"]).to be_nil
+
+        ReactOnRails.configure do |config|
+          config.build_production_command = "a string or a module"
+        end
+
+        expect(ENV["WEBPACKER_PRECOMPILE"]).to eq("false")
+      end
```

Comment:
all the lines above 79-80 with nil and check that the value for the ENV is reset.